### PR TITLE
New test for snapper (bsc#935923)

### PR DESCRIPTION
**Pull request by cwh42**

### CODE REVIEW

File: main.pm [hunk 1]
```diff
@@ -427,6 +427,7 @@ sub load_consoletests() {
     if (consolestep_is_applicable) {
         loadtest "console/consoletest_setup.pm";
         loadtest "console/textinfo.pm";
+	loadtest "console/snapper_snapshots.pm";
```
Comment (aaannz) [inline main.pm]: correct indentation, please

Comment (cwh42) [inline main.pm]: Fixed

**Comment by aaannz** [comment]
Just a question, will this test pass when fs is formatted as i.e. ext4?

**Comment by cwh42** [comment]
Indeed it only works with btrfs.
We expected btrfs to be default - but in fact it's only default if the disk is big enough.
Unfortunately none of the snapper related tests is executed in openQA because snapper_is_applicable() never returns true because it checks the value of FILESYSTEM. Filesystem except for the 'btrfs' tests - but those tests are unfortunately INSTALLONLY, too.

We could use check_screen() in the partitioning dialog to detect if btrfs is used?

**Comment by aaannz** [comment]
Thanks for clarification, didn't know about snapper_is_applicable.
So I'll merge this and we can figure out how to enabled it more in another PR. (Like add check_screen you mentioned and set some pseudo-internal variable e.g. '_FILESYSTEM' and check that)

**Comment by coolo** [comment]
btrfs is always default - snapper is not. And the snapper_is_applicable needs to check for FILESYSTEM being empty I would think

**Comment by nilxam** [comment]
https://openqa.opensuse.org/tests/74738/modules/snapper_snapshots/steps/3

**Pull request merged**